tests/models/test_cli.py

```diff
@@ -413,6 +414,28 @@ def test_prepare_env_fails(sk_model):
         assert p.wait() != 0
 
 
+@pytest.mark.parametrize("enable_mlserver", [True, False])
+def test_generate_dockerfile(sk_model, enable_mlserver):
+    with mlflow.start_run() as active_run:
+        if enable_mlserver:
+            mlflow.sklearn.log_model(
+                sk_model, "model", extra_pip_requirements=[PROTOBUF_REQUIREMENT]
+            )
+        else:
+            mlflow.sklearn.log_model(sk_model, "model")
+        model_uri = "runs:/{run_id}/model".format(run_id=active_run.info.run_id)
+    extra_args = ["--install-mlflow"]
+    if enable_mlserver:
+        extra_args.append("--enable_mlserver")
+
+    output_directory_name = pyfunc_build_image(model_uri, extra_args=extra_args)
```

Comment:
```suggestion
    output_directory_name = pyfunc_generate_dockerfile(model_uri, extra_args=extra_args)
```

Is this line correct? I think we should call `pyfunc_generate_dockerfile` here.

Reply:
Yep, you are correct, my bad :sweat: